neuralmagic/benchmarks/configs/benchmark_serving.json

```diff
@@ -8,6 +8,7 @@
 				"mistralai/Mistral-7B-Instruct-v0.2",
 				"NousResearch/Llama-2-7b-chat-hf"
 			],
+			"use_all_available_gpus" : "",
```

Comment:
This actually translates to `use_all_available_gpus = True` in python.. If we want to set it to false, we remove this key. 

This is really a pain  - I am considering moving to yml or some other format. on top this, JSON doesn't even let you add comments  :(